stdlib/ctypes/_endian.pyi

```diff
@@ -1,11 +1,14 @@
 import sys
-from _ctypes import RTLD_GLOBAL as RTLD_GLOBAL, RTLD_LOCAL as RTLD_LOCAL, Structure, Union
-from ctypes import DEFAULT_MODE as DEFAULT_MODE, cdll as cdll, pydll as pydll, pythonapi as pythonapi
+from _ctypes import Structure, Union
 
-if sys.version_info >= (3, 12):
+if sys.version_info < (3, 13):
+    from _ctypes import RTLD_GLOBAL as RTLD_GLOBAL, RTLD_LOCAL as RTLD_LOCAL
```

Comment:
All of these look like incidental re-exports, can we remove all the `as` imports?

Looks like it used to do `from ctypes import *`.

Reply:
https://github.com/python/typeshed/pull/12372/commits/adb8d1059194c829cae6ea9a1ee03bec85746639

Reply:
Let me also add an ignore for older Python versions.